### DISCUSSION TITLE
Does Ngx-Editor support inline images?

### DISCUSSION THREAD

**Discussion by chobo2** [sibiraj-s/ngx-editor#420] [Q&A] [answered]
Hi

I am looking at the editor and wondering can I paste or drag images into the editor and have them show. I see the image button in the toolbar but that only allows images that have urls to be added which will not work for me.

**Comment by sibiraj-s** [comment, marked as answer]
Hi @chobo2 . The feature is not supported at the moment. You could build a prosemirror plugin to handle that feature. And PR's welcome too.

[a comment by an automated account is not shown]
